nominees/project-konko.json

```diff
@@ -1,7 +1,10 @@
 {
   "name": "Project Konko",
-  "description": "Project Konko is an Open Source Virtual Reality lesson creator built in Unity. It is designed & supported by NubianVR (www.nubianvr.com) and primarily targetted at the Oculus Go VR device - intended to be adapted to other devices down the line. It seeks to provide the tools that will allow developers, creators and educators to: (1)deploy 2D & 360 video lessons in Virtual Reality (2) manipulate 3D object in Virtual Reality environments (3) build multiple choice lesson assessment in Virtual Reality (4) maintain a badge & leaderboard functionality for Virtual Reality lessons.",
```

Comment:
I would preserve all the information we have on this filed

Reply:
Resolved in commit aefad21